doc/architecture/adr/0020-authentication-orm.md

```diff
@@ -0,0 +1,33 @@
+# Authentication ORM
+
+* Status: Accepted
+* Deciders: @MortMH, @Frydensberg1, @CodeReaper, @robertrosborg
+* Date: 2023-02-06
+
+---
+
+## Context and Problem Statement
+
+We need an ORM for the Auth domain to handle entity mapping and database migrations.
```

Comment:
The context should probably mention a wish for simpler handling of data (this was supposed to be an event store) etc.